### ISSUE TITLE
Support for apply --auto-approve flag

### ISSUE THREAD

**Issue by calvinbui** [AzBuilder/terrakube#815]
### Bug description 🐞

Running `terraform apply --auto-approve` causes:

1. CLI to hang 
2. The UI to be on `waitingApproval`, with neither the Approve or Discard buttons working

![image](https://github.com/AzBuilder/terrakube/assets/3604363/1e8b1cb3-337c-4d72-ba40-53f87b535d84)

![image](https://github.com/AzBuilder/terrakube/assets/3604363/4616119f-fce6-42c7-9b62-734368a38343)

To get around this, I can comment out "approval" from the `Terraform-Plan/Apply-Cli` template, but this will affect all CLI runs.

```yaml
flow:
- type: "terraformPlan"
  name: "Terraform Plan from Terraform CLI"
  step: 100
# - type: "approval"
#   name: "Approve Plan from Terraform CLI"
#   step: 150
#   team: "TERRAFORM_CLI"
- type: "terraformApply"
  name: "Terraform Apply from Terraform CLI"
  step: 200
```

### Steps to reproduce

Run `terraform apply --auto-approve` on a workspace with changes.

### Expected behavior

The run is auto-approved and proceeds to the `Terraform Apply` stage

**Comment by alfespa17** [comment]
We need to add support to handle the following parameter in the API when running

```
POST /runs
data.attributes.auto-apply
```

Reference:
https://developer.hashicorp.com/terraform/cloud-docs/api-docs/run#request-body

**Comment by alfespa17** [comment]
This feature was implemented in version 2.20.1-beta.1 if you would like to test
